src/uniprotkb/config/evidenceCodes.ts

```diff
@@ -219,55 +224,58 @@ export const ecoCodeToData = {
   },
   [ecoCode.ND]: {
     manual: true,
-    label: 'Manual assertion based on experiment',
+    label: () => 'Manual assertion based on experiment',
     description: 'No biological data available',
   },
   [ecoCode.IEA]: {
     manual: false,
-    label: 'Manual assertion based on experiment',
+    label: () => 'Manual assertion based on experiment',
     description: 'Inferred from electronic annotation',
   },
   [ecoCode.MI]: {
     manual: true,
-    label: 'Manual assertion inferred from database entries',
+    label: () => 'Manual assertion inferred from database entries',
     description: 'Manually imported',
     labelRender: () => labels.IMPORTED,
   },
   [ecoCode.AI]: {
     manual: false,
-    label: 'Automatic assertion inferred from database entries',
+    label: () => 'Automatic assertion inferred from database entries',
     description: 'Automatically imported',
     labelRender: () => labels.IMPORTED,
   },
   [ecoCode.AA]: {
     manual: false,
-    label: 'Automatic assertion according to rules',
+    label: (evidences: Evidence[]) =>
```

Comment:
I guess it is because of the exception that was mentioned yesterday to reuse the same ECO code for something different. Acknowledging this expection, is there still an issue with this section of the code?
Maybe leaving a comment about the exception and why would be good, for us in the future to know that this is something done on purpose